patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -1,10 +1,59 @@
 --- ../src-base/minecraft/net/minecraft/item/Item.java
 +++ ../src-work/minecraft/net/minecraft/item/Item.java
-@@ -58,10 +58,10 @@
+@@ -1,26 +1,8 @@
+ package net.minecraft.item;
+ 
+ import com.google.common.collect.HashMultimap;
+-import com.google.common.collect.Maps;
+ import com.google.common.collect.Multimap;
+-import java.util.List;
+-import java.util.Map;
+-import java.util.Random;
+-import java.util.UUID;
+-import javax.annotation.Nullable;
+-import net.minecraft.block.Block;
+-import net.minecraft.block.BlockDirt;
+-import net.minecraft.block.BlockDoublePlant;
+-import net.minecraft.block.BlockFlower;
+-import net.minecraft.block.BlockPlanks;
+-import net.minecraft.block.BlockPrismarine;
+-import net.minecraft.block.BlockRedSandstone;
+-import net.minecraft.block.BlockSand;
+-import net.minecraft.block.BlockSandStone;
+-import net.minecraft.block.BlockSilverfish;
+-import net.minecraft.block.BlockStone;
+-import net.minecraft.block.BlockStoneBrick;
+-import net.minecraft.block.BlockWall;
++import net.minecraft.block.*;
+ import net.minecraft.block.state.IBlockState;
+ import net.minecraft.client.util.ITooltipFlag;
+ import net.minecraft.creativetab.CreativeTabs;
+@@ -39,13 +21,7 @@
+ import net.minecraft.inventory.EntityEquipmentSlot;
+ import net.minecraft.nbt.NBTTagCompound;
+ import net.minecraft.potion.PotionEffect;
+-import net.minecraft.util.ActionResult;
+-import net.minecraft.util.EnumActionResult;
+-import net.minecraft.util.EnumFacing;
+-import net.minecraft.util.EnumHand;
+-import net.minecraft.util.EnumHandSide;
+-import net.minecraft.util.NonNullList;
+-import net.minecraft.util.ResourceLocation;
++import net.minecraft.util.*;
+ import net.minecraft.util.math.BlockPos;
+ import net.minecraft.util.math.MathHelper;
+ import net.minecraft.util.math.RayTraceResult;
+@@ -58,10 +34,16 @@
  import net.minecraftforge.fml.relauncher.Side;
  import net.minecraftforge.fml.relauncher.SideOnly;
  
 -public class Item
++import javax.annotation.Nullable;
++import java.util.List;
++import java.util.Map;
++import java.util.Random;
++import java.util.UUID;
```

Comment:
I know, I forgot to check before pushing (I was in a hurry). I'll fix those imports tomorrow.